ResourceManager/Strings.cs

```diff
@@ -27,10 +27,9 @@ public sealed class Strings : Translate
 
 Yes, I allow telemetry!");
 
-        private readonly TranslationString _installGitInstructions = new TranslationString("Install git...");
-        private readonly TranslationString _findGitExecutable = new TranslationString("Find git...");
-        private readonly TranslationString _gitExecutableNotFoundText =
-            new TranslationString("The Git executable could not be located on your system.");
+        private readonly TranslationString _installGitInstructions = new TranslationString("Install Git\r\nGo to GitExtensions wiki");
```

Comment:
`TaskDialogCommandLink` has an overload that takes 3 parameters: `TaskDialogCommandLink(string name, string text, string instruction)`.

```suggestion
        private readonly TranslationString _installGitInstructions = new TranslationString("Install git...");
        private readonly TranslationString _installGitInstructionsDetails = new TranslationString("Click here to read instructions");
```